spec/features/switch_organisation_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rack_session_access/capybara'
+
+describe 'Multiple organisations', type: :feature do
+  let(:other_organisation) { create(:organisation) }
+
+  let(:user) { create(:user, organisation: create(:organisation)) }
+
+  before do
+    sign_in_user user
+  end
+
+  context "when the user doesn't belong to the organisation" do
+    it 'dissallows allows switching to that organisation' do
```

Comment:
Doesn't read well / make sense, but I get it.

Reply:
Agreed, have updated it